api/src/movie/movie.controller.ts

```diff
@@ -16,17 +16,15 @@ import { Request } from 'express';
 import { ListAuthGuard } from '../list/guard/list.guard';
 import { RemoveListFieldsInterceptor } from '../list/interceptor/remove-list-fields.interceptor';
 import { MovieService } from './movie.service';
+import { QueryDto } from './dto/query.dto';
 
 @Controller('movies')
 export class MovieController {
   constructor(private movieService: MovieService) {}
 
   @Get('/')
-  async getMovies(
-    @Query('userId') userId?: string,
-    @Query('listId') listId?: string,
-  ) {
-    return this.movieService.getMovies(userId, listId);
+  async getMovies(@Query() query: QueryDto) {
```

Comment:
Should we have a guard for `/` suchthat only admins can get all movies in the db? Not saying it has to be done now, just wondering.

Reply:
I don't see a reason why we have to limit movie results. In the future, we'll have pagination, if that's what you're worrying about.